src/Schema/Directives/WithCountDirective.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Illuminate\Support\Str;
+use Nuwave\Lighthouse\Execution\DataLoader\RelationCountBatchLoader;
+use Nuwave\Lighthouse\Support\Contracts\DefinedDirective;
+use Nuwave\Lighthouse\Support\Contracts\FieldMiddleware;
+
+class WithCountDirective extends WithRelationDirective implements FieldMiddleware, DefinedDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'SDL'
+"""
+Eager-load the count of an Eloquent relation.
+
+The name of the field is assumed to be `${RELATION}_count`,
```

Comment:
We should remove that part, as `@count` should be used when the field should actually resolve to the count.

Reply:
Got it.